go/vt/mysqlctl/backup.go

```diff
@@ -30,9 +30,9 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/klauspost/pgzip"
 	"golang.org/x/net/context"
 
-	"vitess.io/vitess/go/cgzip"
```

Comment:
Did you check if we can nuke this altogether and remove this package from the codebase?